### DISCUSSION TITLE
What are the most common units reported in SCM Consumption messages?

### DISCUSSION THREAD

**Discussion by mik3y** [bemasher/rtlamr#222] [Q&A]
I'm receiving SCM messages from three meters (power, water, and gas). I'm trying to establish what the units of measure are.

Understanding this can probably vary by region, meter, and provider, are there at least some common/popular units?

For example, I think the power meter is reporting some units less than 1 kWh, but don't have enough usage to be certain.

**Comment by bemasher** [comment]
Best to compare measured results with utility bills, they'll often list quantities and units that should correspond to what you're reading.

Keep in mind that meters often transmit higher precision that can be read on the face of the meter.

**Reply by mik3y** [reply]
Oh certainly, will cross check with my bill; I’m just being impatient as it will take me a few more weeks to get that usage data.. :)

I’m also just a little curious about the landscape in general - surely there isn’t _that_ much variation in units reported? Understanding it would come with a big YMMV, was looking for a starting point like, “most US power meters report kWh/1000”.

(Forgot to say, thanks for this project!)

**Comment by powerjc** [comment]
Looking at physical meter could be helpful. Comparing current reading to the value you're getting. If not a match, a reasonable guess would be it increments with the K factor pulse rate of the meter. Typically 7.2Wh or ~139 per kWh on residential electric in the US but worth verifying on the face of your meter.

**Reply by ninjasstudio** [reply]
My water meter weirdly transmits in 10 gallon units, NC, USA
